README.md

```diff
@@ -71,7 +77,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+friends.each do |name|
+  puts "Belle is friends with " + name
+end
 ```
```

Comment:
:+1:

(Although I would use `friend` instead of `name` to represent each item in the array)